tests/common/HtmlTest.php

```diff
@@ -882,12 +882,6 @@ public function testAddCssClass(): void
         Html::addCssClass($options, ['test3']);
         $this->assertSame(['class' => ['test', 'test2', 'test3']], $options);
 
-        $options = [
-            'class' => 'test',
-        ];
-        Html::addCssClass($options, ['test1', 'test2']);
-        $this->assertSame(['class' => 'test test1 test2'], $options);
```

Comment:
Maybe leave the test with new result?
`$this->assertSame(['class' => ['test', 'test1', 'test2']], $options);`

Reply:
Added as separate test: https://github.com/yiisoft/html/pull/171/files#diff-2bb921927af1a632486e414d8a3f66c516f7fdde7b122a677eb2f4183c0d6e27R908